package.json

```diff
@@ -99,10 +97,10 @@
     "zone.js": "~0.11.4"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "^13.0.1",
-    "@angular-devkit/build-ng-packagr": "^0.1100.0-next.5",
+    "@angular-devkit/build-angular": "^13.3.7",
+    "@angular-devkit/core": "^14.0.1",
```

Comment:
Version 14 of devkit core seems fine. I didn't want to pin to anything older than necessary.